### PR TITLE
Amend according to TA's advise

### PR DESCRIPTION
**Pull request by froststein**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -333,13 +333,18 @@ The general workflow of the `session /edit` command is as follows:
...
 7. `SessionEditCommand#run` will check if there is an update for a new list of persons, new session or new session date.
-   * If there is an update on the list of persons. It would first check if the newly provided list of persons contains duplicated names.
-     * If duplicated names are detected, an error message would be printed and control is returned to `SplitLah`.
-     * Else, it would call `PersonList#isSuperSet` to check if the newly supplied list of persons contain all existing persons in the session.
-     * If `PersonList#isSuperSet` returns `false`, an error message would be printed and control is returned to `SplitLah`.
-     * Else, if it returns `true` it would then call `Session#addPerson` to add in the new list of persons.
-   * If there is an update on the session name, `Session#setSessionName` is called to set the new session name.
+   * If there is an update on the list of persons, `SessionEditCommand#getNewPersonList` is called to return a new list of persons to be stored. 
+     * The method would check if the newly provided list of persons contains duplicated names.
```
Comment (matheril) [inline docs/DeveloperGuide.md:337]: Consider changing "would check" to "checks" as future tense doesn't seem necessary here.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -333,13 +333,18 @@ The general workflow of the `session /edit` command is as follows:
...
+       * If duplicated names are detected, an exception is thrown, an error message would be printed and control is returned to `SplitLah`.
```
Comment (matheril) [inline docs/DeveloperGuide.md:338]: Consider changing "would be printed" to "is printed".

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -333,13 +333,18 @@ The general workflow of the `session /edit` command is as follows:
...
+       * Else, it would call `PersonList#isSuperSet` to check if the newly supplied list of persons contains all existing persons in the session.
```
Comment (matheril) [inline docs/DeveloperGuide.md:339]: Consider changing "would call" to "calls".

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -333,13 +333,18 @@ The general workflow of the `session /edit` command is as follows:
...
+       * If `PersonList#isSuperSet` returns `false`, an exception is thrown, an error message would be printed and control is returned to `SplitLah`.
+       * Else, a new list of persons ready to be stored in the session is returned
```
Comment (matheril) [inline docs/DeveloperGuide.md:341]: This line is missing a full stop at the end.

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -515,6 +520,11 @@ The general workflow of the `activity /list` command is as follows:
 ### Add a group
 **API reference:** [`GroupCreateCommand.java`](https://github.com/AY2122S2-CS2113T-T10-1/tp/blob/master/src/main/java/seedu/splitlah/command/GroupCreateCommand.java)
 
+The sequence diagram for GroupCreateCommand is omitted as it bears many similarities with [`SessionCreateCommand`](#add-a-session).
+The interactions of GroupCreateCommand with `Profile` and `Storage` classes are identical but the key differences lie in the arguments being parsed.
+The `GroupCreateCommand` parses only the name and list of persons to store as compared to `SessionCreateCommand` and it creates a new `Group`
```
Comment (matheril) [inline docs/DeveloperGuide.md:525]: Minor, but consider splitting this line into two bullet points: one for each command. Maybe something like

The interactions of GroupCreateCommand with `Profile` and `Storage` classes are identical but the key differences lie in the arguments being parsed:

- `GroupCreateCommand` parses only the name and list of persons. It then creates a new `Group` object and adds it to the list of groups managed by the `Profile` class.
- `SessionCreateCommand` parses the x, y, z.... It then creates a new .... and stores it in .....

But you can choose not to do this if it seems overly repetitive. In that case, do try to split the sentence up into two like the first bullet point to make it easier to follow.

**Review by matheril — CHANGES_REQUESTED** [review]
Minor changes requested

**Review by warrencxw — APPROVED** [review]
Looks good to merge.

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -90,9 +90,9 @@ The diagram depicts a scenario where a user attempts to create a session.
 ![SplitLah Component Screenshot](https://raw.githubusercontent.com/AY2122s2-cs2113t-t10-1/tp/master/docs/images/developerguide/SplitLahComponent.drawio.png)
 <br>
 The `SplitLah` component is the application's main class. Its job is to initialize an instance of `Manager` when the
-application starts. After initialization, it would then proceed to run a loop which would prompt the user for a 
-command. When it receives a command from the user, it would invoke the `parser` and retrieve the command for SplitLah
-to run. Upon using the `Exit` command, SplitLah would then exit from the command loop and end the application.
+application starts. After initialization, it proceeds to run a loop which prompts the user for a 
+command. When it receives a command from the user, it invokes the `parser` and retrieve the command for SplitLah
```
Comment (matheril) [inline docs/DeveloperGuide.md:94]: Consider capitalising the P in `parser`.

**Review by matheril — CHANGES_REQUESTED** [review]
Looks good to merge after making a minor capitalisation change

**Review by matheril — CHANGES_REQUESTED** [review]
Looks good to merge

**Review by matheril — APPROVED** [review]
Looks good to merge

**Pull request merged**